db/migrate/20181102141329_add_ssl_options_to_endpoints.rb

```diff
@@ -0,0 +1,6 @@
+class AddSslOptionsToEndpoints < ActiveRecord::Migration[5.1]
+  def change
+    add_column :endpoints, :verify_ssl,            :integer
```

Comment:
Should this be a boolean?

Reply:
Oh hm, that adds more logic on the provider to have to check if cert_authority is set because before you could pass verify_ssl in to the native client but I guess it makes it easier on the API